### PR TITLE
Support for Swagger 2.0

### PR DESCRIPTION
**Pull request by xl32**

### CODE REVIEW

File: src/SwaggerModule/Module.php [hunk 1]
```diff
@@ -62,10 +62,26 @@ public function getServiceConfig()
...
                 },
 
-                'Swagger\Swagger' => function($serviceManager) {
+                'Swagger\Annotations\Swagger' => function($serviceManager) {
                     /** @var $options \SwaggerModule\Options\ModuleOptions */
                     $options = $serviceManager->get('SwaggerModule\Options\ModuleOptions');
-                    return new SwaggerLibrary($options->getPaths());
+                    $analyser = new \Swagger\StaticAnalyser();
+                    $analysis = new \Swagger\Analysis();
+                    $processors = \Swagger\Analysis::processors();
```
Comment (davidwindell) [inline src/SwaggerModule/Module.php]: Please add `use` statements for these.

File: src/SwaggerModule/Module.php [hunk 2]
```diff
@@ -62,10 +62,26 @@ public function getServiceConfig()
...
+
+                    // Crawl directory and parse all files
+                    $paths = $options->getPaths();
+                    foreach($paths as $directory) {
+                        $finder = \Swagger\Util::finder($directory);
```
Comment (davidwindell) [inline src/SwaggerModule/Module.php]: And this.

File: src/SwaggerModule/Module.php [hunk 3]
```diff
@@ -62,10 +62,26 @@ public function getServiceConfig()
                     return new SwaggerModuleOptions($config);
                 },
 
-                'Swagger\Swagger' => function($serviceManager) {
+                'Swagger\Annotations\Swagger' => function($serviceManager) {
```
Comment (davidwindell) [inline src/SwaggerModule/Module.php]: We should probably create a factory for this but that can wait for another PR.

File: src/SwaggerModule/Module.php (hunk 1 above)
Comment (xl32) [inline src/SwaggerModule/Module.php]: Done

File: src/SwaggerModule/Module.php (hunk 2 above)
Comment (xl32) [inline src/SwaggerModule/Module.php]: Done

File: src/SwaggerModule/Module.php [hunk 4]
```diff
@@ -21,8 +21,11 @@
 namespace SwaggerModule;
 
 use RuntimeException;
-use Swagger\Swagger as SwaggerLibrary;
+use Swagger\Annotations\Swagger as SwaggerLibrary;
```
Comment (davidwindell) [inline src/SwaggerModule/Module.php]: Remove this line, SwaggerLibrary no longer required?

Comment (xl32) [inline src/SwaggerModule/Module.php]: Done.

**Pull request merged**